tests/e2e/BasicTest.php

```diff
@@ -162,4 +186,26 @@ protected function assertCreatedNewSession(Response $response)
         $this->assertTrue($response->hasHeader('Set-Cookie'));
         $this->assertStringStartsWith('PHPSESSID=', $response->getHeaderLine('Set-Cookie'));
     }
+
+    /**
+     * Asserts whether a received request triggered the creation of a new session with example custom parameters.
+     * It does so by searching for and inspecting the 'Set-Cookie' header.
+     * 
+     * Expected result:
+     * set-cookie:PHPSESSID=0691f472784601a324eadf78c05cf4c1; expires=Tue, 11-Jul-2017 14:58:34 GMT; Max-Age=86400; path=/; secure; HttpOnly
+     *
+     * @param Response $response
+     */
+    protected function assertCustomCookieParams(Response $response)
+    {
+        $this->assertTrue($response->hasHeader('Set-Cookie'));
+        $this->assertStringStartsWith('PHPSESSID=', $response->getHeaderLine('Set-Cookie'));
+        $this->assertRegexp('/expires=/i', $response->getHeaderLine('Set-Cookie'));
+        $this->assertRegexp('/path=/i', $response->getHeaderLine('Set-Cookie'));
+        $this->assertRegexp('/secure/i', $response->getHeaderLine('Set-Cookie'));
+        $this->assertRegexp('/HttpOnly/i', $response->getHeaderLine('Set-Cookie'));
```

Comment:
Just one little thing: PHPStorm is whining that the actual method's name is `assertRegExp`, with a capital E.